<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="COG-WIN-MAC03" failures="2" tests="15" name="testcases.TicketListingTest" time="2704.721" errors="0" timestamp="2022-08-31T18:09:26 IST" skipped="0">
  <testcase classname="testcases.TicketListingTest" name="verifyUrlOnClickingBreadCrumb" time="58.434"/>
  <system-out/>
  <testcase classname="testcases.TicketListingTest" name="verifyResetButtonFunctionality" time="310.425"/>
  <system-out/>
  <testcase classname="testcases.TicketListingTest" name="verifyExpandCollapseFeature" time="68.042"/>
  <system-out/>
  <testcase classname="testcases.TicketListingTest" name="verifySearchFunctionality" time="302.544"/>
  <system-out/>
  <testcase classname="testcases.TicketListingTest" name="verifyUnassignedTicketWidget" time="73.559"/>
  <system-out/>
  <testcase classname="testcases.TicketListingTest" name="verifyReplyCount" time="258.349"/>
  <system-out/>
  <testcase classname="testcases.TicketListingTest" name="verifyFollowUpFunctionality" time="276.201">
    <failure message="Select Date picker -  element not clikabke" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Select Date picker -  element not clikabke
at org.testng.Assert.fail(Assert.java:99)
at utils.WebBasePage.click(WebBasePage.java:91)
at pageobjects.TicketListingPage.clickOnDatePicker(TicketListingPage.java:287)
at action.TicketListingAction.verifyFollowUp(TicketListingAction.java:128)
at testcases.TicketListingTest.verifyFollowUpFunctionality(TicketListingTest.java:127)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifyFollowUpFunctionality -->
  <system-out/>
  <testcase classname="testcases.TicketListingTest" name="verifyMergeButtonFunctionality" time="77.647"/>
  <system-out/>
  <testcase classname="testcases.TicketListingTest" name="verifyUpdateTicketFunctionality" time="260.666"/>
  <system-out/>
  <testcase classname="testcases.TicketListingTest" name="verifyDeleteTicketFunctionality" time="123.831">
    <failure message="java.lang.NullPointerException: Cannot invoke &quot;org.openqa.selenium.WebElement.clear()&quot; because &quot;element&quot; is null" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: java.lang.NullPointerException: Cannot invoke "org.openqa.selenium.WebElement.clear()" because "element" is null
at org.testng.Assert.fail(Assert.java:99)
at utils.WebBasePage.enter(WebBasePage.java:64)
at pageobjects.UserLoginPage.enterUserName(UserLoginPage.java:77)
at pageobjects.UserLoginPage.forcefulLogOutLogin(UserLoginPage.java:123)
at action.LoginAction.logoutLogin(LoginAction.java:26)
at testcases.TicketListingTest.verifyDeleteTicketFunctionality(TicketListingTest.java:169)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifyDeleteTicketFunctionality -->
  <system-out/>
  <testcase classname="testcases.TicketListingTest" name="verifyNotesFunctionality" time="244.664"/>
  <system-out/>
  <testcase classname="testcases.TicketListingTest" name="verifyAttachmentTabFunctionality" time="187.176"/>
  <system-out/>
  <testcase classname="testcases.TicketListingTest" name="verifyWidgetsFunctionality" time="77.003"/>
  <system-out/>
  <testcase classname="testcases.TicketListingTest" name="verifyCannedReplyFunctionality" time="189.087"/>
  <system-out/>
  <testcase classname="testcases.TicketListingTest" name="verifyDraftReplyFunctionality" time="197.093"/>
  <system-out/>
</testsuite> <!-- testcases.TicketListingTest -->
